charger/easee.go

```diff
@@ -500,3 +507,40 @@ func (c *Easee) Identify() (string, error) {
 
 	return c.rfid, nil
 }
+
+// Set smart charging status to update the chargers led (smart=blue, fast=white)
+func (c *Easee) updateSmartCharging() {
+	if c.lp == nil {
+		return
+	}
+
+	mode := c.lp.GetMode()
+	isSmartCharging := mode == api.ModePV || mode == api.ModeMinPV
+
+	c.mux.L.Lock()
+	updateNeeded := isSmartCharging != c.smartCharging
+	c.mux.L.Unlock()
+
+	if updateNeeded {
+		c.log.DEBUG.Printf("update smart charging status: %v", isSmartCharging)
```

Comment:
Jedesmal loggen für eine nice to have Funktion? Wenns nicht tut siehst dus am Fehler?